argo/src/Argo/DefaultMain.hs

```diff
@@ -110,22 +110,25 @@ data ProgramMode stdioOpts socketOpts httpOpts docOpts
 data GlobalOptions stdioOpts socketOpts httpOpts docOpts =
     GlobalOptions
     { methodOpts :: MethodOptions
+     -- ^ Max number of live concurrent states the server supports.
```

Comment:
Isn't there more to `MethodOptions` that just the maximum number of live concurrent states? I feel like this comment ought to be a bit more general that just that.

Reply:
Oops, yes that doesn't make sense. That's a copy+paste / code-reorg error on my part as I was experimenting with where to place things. Thanks!